test/functional/rpc_blockchain.py

```diff
@@ -410,6 +410,9 @@ def _test_getblock(self):
         self.log.info("Test that getblock with verbosity 2 still works with pruned Undo data")
         datadir = get_datadir_path(self.options.tmpdir, 0)
 
+        self.log.info("Test that getblock with invlaid verbosity type returns proper error message")
```

Comment:
```suggestion
        self.log.info("Test that getblock with invalid verbosity type returns proper error message")
```

Reply:
My apologies for that. Have updated the PR to fix the typos. Thanks!